### ISSUE TITLE
source that doesn't work

### ISSUE THREAD

**Issue by mdsumner** [mdsumner/ncdf4#1]
HELP!  Does anyone know why this source doesn't work?  (rwinlib is at 4.4.1.1)

https://cida.usgs.gov/thredds/dodsC/stageiv_combined

It's in this catalog: 

https://cida.usgs.gov/thredds/catalog.html

**Comment by dblodgett-usgs** [comment]
You can get a subset file with nccopy.

10 time steps with:
`nccopy https://cida.usgs.gov/thredds/dodsC/stageiv_combined?Total_precipitation_surface_1_Hour_Accumulation[0:1:10][0:1:880][0:1:1120],lon[0:1:1120][0:1:880],lat[0:1:1120][0:1:880],time[0:1:10] test.nc`

<img width="756" alt="screen shot 2018-08-20 at 8 07 32 pm" src="https://user-images.githubusercontent.com/1492803/44374613-cae6ec00-a4b4-11e8-865b-4db4ea77ccd0.png">

That said, this dataset is a total edge case. 

I would be curious if a more typical dataset `nc_open`s? `https://cida.usgs.gov/thredds/dodsC/prism_v2` ?

**Comment by mdsumner** [comment]
Yes, that one works fine  - also no problem opening the test.nc locally fwiw
